aries-site/src/tests/navigation/deeplinks.js

```diff
@@ -0,0 +1,52 @@
+
+/* eslint-disable no-undef */
+import { ReactSelector, waitForReact } from 'testcafe-react-selectors';
+import { Selector } from 'testcafe';
+import { baseUrl } from '../utils';
+
+// A Deeplink is a subsection of the sections. This is testing if the page is taken to the right section of page to match the hash
+
+// Selector name found using Chrome React dev tools on prod mode of website
+const Search = 'D Box';
+
+const getSuggestion = page => {
```

Comment:
Should Search and getSuggestion be pulled into utils if it's being used across files?

Reply:
I kinda thought the same since its being used now in 2 places where as before just one. I can move it